01-lets_get_started/01.cpp

```diff
@@ -5,34 +5,41 @@ using namespace std;
 
 class square
 {
-    private:
-        int side,column,row;
-    public:
-       void square_data(){
-         cout << "Enter side of Square\n";
-         cin >> side;
-         top_bottom();
-         if(side > 1){
-          cout << "\n";
-          left_right();
-          top_bottom();
-         }
-        }
-       void top_bottom(){
-          for(column = 0; column < side; ++column)
-            cout << "#";
-         }
-        void left_right(){
-          for(row = 0; row < side-2; ++row){
-            cout<< "#";
-          for(column=0; column < side-2; ++column)
-            cout << " ";
-          cout << "#\n";
-         }
-        }
+private:
+  int side,column,row;
+public:
+  void square_data()
+  {
+    cout << "Enter side of Square\n";
+    cin >> side;
+    top_bottom();
+    if(side > 1)
+    {
+      cout << "\n";
+      left_right();
+      top_bottom();
+    }
+  }
+  void top_bottom()
+  {
+    for(column = 0; column < side; ++column)
+    cout << "#";
```

Comment:
Consider using a constant to represent `#`; define it at the top, it will make the program more readable.